ghcide/src/Development/IDE/Import/FindImports.hs

```diff
@@ -143,10 +155,12 @@ locateModule env comp_info exts targetFor modName mbPkgName isSource = do
     Nothing -> do
 #endif
 
-      mbFile <- locateModuleFile ((homeUnitId_ dflags, importPaths dflags) : other_imports) exts targetFor isSource $ unLoc modName
+      mbFile <- locateModuleFile ((homeUnitId_ dflags, importPaths dflags, S.empty) : other_imports) exts targetFor isSource $ unLoc modName
```

Comment:
This needs a comment explaining why `S.empty` is correct here.

Reply:
added